queue_services/business-pay/src/business_pay/resources/pay_filer.py

```diff
@@ -174,17 +171,38 @@ def publish_to_filer(filing: Filing, payment_token: PaymentToken):
         logger.debug(
             f"checking filer for pay-id: {payment_token.id} on filing: {filing}")
         if filing.effective_date <= filing.payment_completion_date:
-            filer_topic = current_app.config["FILER_PUBLISH_OPTIONS"]["subject"]
-            queue_message = create_filing_msg(filing.id)
-            logger.debug(f"filer queue_message: {queue_message}")
-
             try:
-                # await queue.publish(subject=filer_topic, msg=queue_message)
-                queue.publish_json(subject=filer_topic, payload=queue_message)
+                subject = current_app.config.get('BUSINESS_FILER_TOPIC')
+                data = create_filing_msg(filing.id)
+
+                ce = SimpleCloudEvent(
+                    id=str(uuid.uuid4()),
+                    source='business-pay',
+                    subject=subject,
+                    time=datetime.now(timezone.utc),
+                    type='filing-message',
+                    data = data
+                )
+                gcp_queue.publish(subject, to_queue_message(ce))
+                logger.debug(
+                    f"Publish Event to Filer: {str(ce)}"
+                )
             except Exception as err:
                 logger.debug(
                     f"Publish to Filer error: {err}, for pay-id: {payment_token.id}")
 
+            # TODO: remove NATS publishing
+            # filer_topic = current_app.config["FILER_PUBLISH_OPTIONS"]["subject"]
+            # queue_message = create_filing_msg(filing.id)
+            # logger.debug(f"filer queue_message: {queue_message}")
+
+            # try:
+            #     # await queue.publish(subject=filer_topic, msg=queue_message)
+            #     queue.publish_json(subject=filer_topic, payload=queue_message)
+            # except Exception as err:
+            #     logger.debug(
+            #         f"Publish to Filer error: {err}, for pay-id: {payment_token.id}")
+
```

Comment:
We cannot merge back to main If we comment this. Can you change this code in such a way that if the `enable-sandbox` FF is true then publish to gcp else nats.

PS: I created `enable-sandbox` FF yesterday.

Reply:
Updated.

I don't import Flags service from legal-api because there're lots of dependency conflicts and I assume we don't want to downgrade Flask3 in business-pay. So I created a new serivce in this component with some tweaks of the original legal-api one, since LaunchDarkly SDK of business-pay is also in higher version.

Reply:
FYI about Flask3 updates if anyone has the error like `app.env does not exist` etc. in the future
![image](https://github.com/user-attachments/assets/d8218974-755f-4e58-acb9-038da00c6b35)